wait/http_test.go

```diff
@@ -48,8 +52,8 @@ func TestHTTPStrategyWaitUntilReady(t *testing.T) {
 		return
 	}
 
-	capath := workdir + "/testdata/root.pem"
-	cafile, err := ioutil.ReadFile(capath)
+	capath := path.Join(workdir, "testdata", "root.pem")
```

Comment:
I think filepath will be more robust across different OS. See https://stackoverflow.com/a/70539540/2382761

Reply:
resolved via [3358132](https://github.com/testcontainers/testcontainers-go/pull/569/commits/3358132ab900cd3f5b98e3d4047d491642bd2d82)